pages/linux/hostid.md

```diff
@@ -0,0 +1,7 @@
+# hostid
+
+> Prints the numeric identifier for the current host (not necessarily the IP address).
+
+- Display the numeric identifier for the current host (in hexadecimal):
```

Comment:
No need for the parenthesis.

Reply:
I understand, but it might help someone in preventing subtle bugs.
The rest of the description is already pretty short.

Reply:
I mean that it returns the IP address in most cases. But might return something else if it is configured in such a way. See the links in my references comment above.

Reply:
OK, but how do I do that? Go to *Files Changed*, click on *Edit* and make a new commit on that page?

Reply:
Yes. Or if you are using the CLI client, you can edit the file, make a commit and push it.

Reply:
Just to confirm, this will be the new sentence:

    > Prints the numeric identifier for the current host, which need not be the IP address in all cases.

Reply:
Umm no. I wasn't talking about the heading. I will do it myself.